src/applications/claims-status/components/ClaimLetterList.jsx

```diff
@@ -3,11 +3,16 @@ import PropTypes from 'prop-types';
 
 import ClaimLetterListItem from './ClaimLetterListItem';
 
-const ClaimLetterList = ({ letters }) => {
-  return letters.map(letter => (
+const renderLetters = letters =>
+  letters.map(letter => (
     <ClaimLetterListItem key={letter.documentId} letter={letter} />
   ));
-};
+
+const ClaimLetterList = ({ letters }) => (
+  <ol className="vads-u-margin--0 vads-u-padding--0">
```

Comment:
Minor suggestion here…
- Add "usa-unstyled-list" to the className.
- Remove `vads-u-display--block` from [this line](https://github.com/department-of-veterans-affairs/vets-website/pull/22638/files#diff-b58c5b99ec670a67b0a3fbe23d63937a6264379c9a864a7a91088eefac3252d6R17)
- Add `role="list"`. ESLint will complain about redundant roles, so you might need to ignore it. Basically, this is a bug with VoiceOver+Safari where list semantics get ignored with either `list-style: none` or changing display properties of list items, though some might say feature, that list semantics get ignored. 

Adding the `role` attribute will fix a bug that definitely exists in Safari. Updating the class names makes their design intent a little easier to grok.